src/cdk/drag-drop/drag.ts

```diff
@@ -491,6 +491,19 @@ export class CdkDrag<T = any> implements AfterViewInit, OnDestroy {
       // Preserve the previous `transform` value, if there was one.
       this._rootElement.style.transform = this._initialTransform ?
           this._initialTransform + ' ' + transform : transform;
+
+      // Apply transform as attribute if dragging and svg element to work for IE
+      if (this._rootElement instanceof SVGElement) {
+        let appliedTransform = getComputedStyle(this._rootElement).getPropertyValue('transform');
+        // Check if the value is a matrix3d(), and convert to matrix() for IE compatibility
+        const match = appliedTransform.match(/^matrix3d\((.+)\)$/);
```

Comment:
I don't think using a matrix here is necessary. From what I can tell, IE supports stacking the SVG transforms similarly to the CSS ones. E.g. note how the blue square has `translate(50) translate(-25)` here: https://s.codepen.io/crisbeto/debug/qQpqrQ/ZoABaKZbgybr

Reply:
You're right, `translate()` works, and greatly simplifies things.  Changed.